pkg/kvstore/allocator/allocator.go

```diff
@@ -710,15 +702,15 @@ func (a *Allocator) RunGC() error {
 
 		// fetch list of all /value/<key> keys
 		valueKeyPrefix := path.Join(a.valuePrefix, string(v))
-		uses, err := kvstore.ListPrefix(valueKeyPrefix)
+		k, v, err := kvstore.GetPrefix(context.Background(), valueKeyPrefix)
```

Comment:
This is not correct, see #8172. #8172 fixes this for both `GetNoCache()` and `RunGC()`.

Reply:
I'll drop this commit